pyproject.toml

```diff
@@ -10,19 +10,16 @@ readme = "README.md"
 python = "^3.11"
 cryptography = "^42.0.5"
 flask = "^3.0.2"
-flask-limiter = { extras = ["redis"], version = "^3.7.0" }
 flask-migrate = "^4.0.7"
 flask-sqlalchemy = "^3.1.1"
 flask-wtf = "^1.2.1"
-gunicorn = "^22.0.0"
```

Comment:
I don't think we want to remove gunicorn

Reply:
yeah, that was not intentional

Reply:
I see the merge commit but diff still shows this as removed